<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProverkaDialog</class>
 <widget class="QDialog" name="ProverkaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить строку</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="tipKanalaLabel">
       <property name="text">
        <string>Тип канала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tipKanalaComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Вентиляционный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дымовой</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="materialKanalaLabel">
       <property name="text">
        <string>Материал канала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="materialKanalaComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кирпич</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оцинкованная сталь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Нержавеющая сталь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Коаксиальный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Металл</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="diametrKanalaLabel">
       <property name="text">
        <string>Сечение или диаметр канала, мм</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="diametrKanalaComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø40/100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø60/100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø110/150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø115</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø120</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø130</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø135</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø180</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ø200</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="gazoispOborudovanieLabel">
       <property name="text">
        <string>Газоиспользующее оборудование или помещение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gazoispOborudovanieComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Котел</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Теплогенераторная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Водонагреватель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кухня</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="sostoyanieKanalaLabel">
       <property name="text">
        <string>Состояние канала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sostoyanieKanalaComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Очищен</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Не очищен</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="nalichieTyagiLabel">
       <property name="text">
        <string>Наличие тяги и соответствие нормативным требованиям</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="nalichieTyagiComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Соответствует</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Не соответствует</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="sostojanieOgolovkaLabel">
       <property name="text">
        <string>Состояние оголовка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sostojanieOgolovkaComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Исправен</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Не исправен</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Удовлетворительно</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Не удовлетворительно</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/ok32.png</normaloff>:/images/ok32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
